lib/destination/ddl/expiry.go

```diff
@@ -5,11 +5,22 @@ import (
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/artie-labs/transfer/lib/config/constants"
 )
 
 func ShouldDeleteFromName(name string) bool {
+	// We expect the table name to be in the format of `tableName__artie_suffix_unix`
+	if !strings.Contains(strings.ToLower(name), constants.ArtiePrefix) {
+		return false
+	}
+
 	nameParts := strings.Split(name, "_")
-	if len(nameParts) < 2 {
+	if len(nameParts) < 6 {
```

Comment:
Why 6?

Reply:
That's just how many are in a temp table

Reply:
At the minimum

Reply:
I could put it back to 2 if it's confusing?

Reply:
Oh I see, because right after the table name there are 3 underscores in a row, so 2 of the 6 parts are empty. Maybe it would be more clear to just split the part that comes after the `constants.ArtiePrefix` and check for 2 parts there? Because otherwise if the table name itself has any underscores in it, we could get a false positive here.